doc/rados/operations/crush-map-edits.rst

```diff
@@ -313,6 +313,13 @@ and one rack bucket. The OSDs are declared as items within the host buckets::
 	   fairly “compete” against each other for replica placement through a
 	   process analogous to a draw of straws.
 
+        #. **Straw2**: List and Tree buckets improves Straw that correctly
```

Comment:
Straw2 buckets improve Straw to correctly avoid any data movement between items when neighbor weights change.